src/OrbitGl/TrackManager.cpp

```diff
@@ -170,15 +174,19 @@ void TrackManager::SetFilter(const std::string& filter) {
 void TrackManager::UpdateVisibleTrackList() {
   visible_tracks_.clear();
 
+  auto track_should_be_shown = [this](const Track* track) {
```

Comment:
this function is lowercase because it is a lambda function?

Reply:
Yes. It's not a function definition, but a local variable. Even though I agree that it looks weird.